bench/security/common.h

```diff
@@ -7,4 +7,9 @@
 
 #define NOT_CAUGHT() do { puts("NOT_CAUGHT"); fflush(stdout); } while ((0));
 
+__attribute((noinline))
```

Comment:
We might want to add some `#ifdef` to accommodate various compilers (or simply `__has_attribute(noinline))`):

- [`__declspec(noinline)`]( https://docs.microsoft.com/en-us/cpp/cpp/noinline?view=vs-2019) for Visual Studio
- [`__attribute__((noinline))`](http://gcc.gnu.org/onlinedocs/gcc-4.4.4/gcc/Function-Attributes.html) for gcc, or `[[gnu::noinline]]`
- [`#pragma noinline`](https://www.intel.com/content/www/us/en/develop/documentation/oneapi-dpcpp-cpp-compiler-dev-guide-and-reference/top/compiler-reference/pragmas/intel-specific-pragma-reference/inline-noinline-forceinline.html) for ICC
- Maybe pass `-fno-inline -fno-builtin-inline -fno-inline-small-functions` as well?